backend/data/processors/manual_fixes/MATHFixes.py

```diff
@@ -165,10 +244,13 @@ def MATH_3611_3701(condition):
     """
     "original": "Prerequisite: 12 UOC of Level 2 Mathematics with an average mark of at least 70, including MATH2111 or MATH2011 (CR) or MATH2510 (CR), or permission from the Head of Department.<br/><br/>",
 
-    "processed": "12UOC in L2 Mathematics with an average 70GRADE && MATH2111 || 65GRADE in MATH2011 || 65GRADE in MATH2510 || permission from the Head of Department" 
-    
+    "processed": "12UOC in L2 Mathematics with an average 70GRADE && MATH2111 || 65GRADE in MATH2011 || 65GRADE in MATH2510 || permission from the Head of Department"
+
     "warning": "Can circumvent prerequisites with permission from Head of Department"
     """
+
+    # TODO: JOEL: This doesn't actually work?? It doesn't like index 9 'MATH'
+    # I think it just doesn't like '70GRADE in LX XXXX' since it's 2 words. So what's the fix?
```

Comment:
this should work with the CourseLevelCategory() class (or similar) - look for it in create.py and category.py. If we arent generating that properly, that should probably be fixed.

Reply:
(in a new card, so make a new bug card here)